Take 2.1/Assets/Scripts/Controls.cs

```diff
@@ -57,13 +64,21 @@ void Update()
     private void FixedUpdate()
     {
         //flips sprite on x-axis when changing direction
-        if (inputX > 0)
-            transform.localScale = new Vector3(-1.0f, 1.0f, 1.0f);
-        if (inputX < 0)
+        if (inputX > 0.1f)
             transform.localScale = new Vector3(1.0f, 1.0f, 1.0f);
+        if (inputX < 0.1f)
```

Comment:
doesn't this mean that the player will move left if nothing is being input as doesn't 'getAxisRaw' produce either: = -1, 0 or 1?